tests/files/FileproxyTest.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+import os
+
+from coalib.files.Fileproxy import Fileproxy
+
+
+class FileproxyTest(unittest.TestCase):
+
+    def setUp(self):
+        fileproxy_test_dir = os.path.join(os.path.split(__file__)[0],
+                                          'FileproxyTestFiles')
+
+        self.testfile = os.path.join(fileproxy_test_dir, 'testfile.txt')
+        self.fileproxyobject = Fileproxy(self.testfile)
+
+    def test_str(self):
+        self.assertEqual(str(self.fileproxyobject),
+                         open(self.testfile).read())
+
+    def test_equal(self):
+        with open(self.testfile, 'a+') as file:
+            lines = file.readlines()
+            file.write("New line")
```

Comment:
tests shouldn't modify persistent files on the filesystem. If they do, use tempfiles and clean them up properly on teardown.